jsonnet/main.jsonnet

```diff
@@ -172,6 +172,9 @@ removeLimits(
   { ['prometheus-user-workload/' + name]: kp.prometheusUserWorkload[name] for name in std.objectFields(kp.prometheusUserWorkload) } +
   { ['prometheus-adapter/' + name]: kp.prometheusAdapter[name] for name in std.objectFields(kp.prometheusAdapter) } +
   { ['grafana/' + name]: kp.grafana[name] for name in std.objectFields(kp.grafana) } +
+  // needs to be removed once 4.3 ships, as this is needed for removal of the
```

Comment:
is this still relevant, as 4.3 sort of is shipped and this is landing in 4.4

Reply:
Good catch, this should now be 4.4.

Reply:
Fixed.